from django.core.exceptions import ValidationError
from django.utils.deconstruct import deconstructible

def validate_report_probability_ranking(structs: dict):
    int

